maven-archetype-plugin/src/it/build-archetype-and-run-its-with-existing-parent/src/test/resources/projects/basic/verify.groovy

```diff
@@ -0,0 +1 @@
+System.out.println(" Yeah Baby it rocks!")
```

Comment:
Please add trailing line break and change to proper English: "Yeah baby, it rocks!"